tests/renderer/components/__snapshots__/commands-spec.tsx.snap

```diff
@@ -28,6 +28,11 @@ exports[`Commands component renders 1`] = `
       />
     </Blueprint3.ControlGroup>
   </div>
+  <div
+    className="title"
+  >
+    Electron Fiddle
+  </div>
```

Comment:
```suggestion
```

Revert.

see: https://github.com/electron/fiddle/blob/28ec7c77d7276f9b8e8ba28c700ff39f7e35ba45/src/renderer/components/commands.tsx#L60-L63

CI machine is `linux`, not `macOS`. This will cause the unit tests in the CI to fail

Reply:
Ok, I've made some changes